lib/manager/components/deployment/CustomConfig.js

```diff
@@ -65,39 +96,48 @@ export default class CustomConfig extends Component<{
   render () {
     const {deployment, name, label} = this.props
     const useCustom = deployment[name] !== null
+    const useCustomUrl = deployment[name + 'Url'] !== null
     const value = this.state[name] || deployment[name]
+    const urlValue = this.state[name + 'Url'] || deployment[name + 'Url']
     const validJSON = isValidJSONC(value)
+    const validURL = !!urlValue && validator.isURL(urlValue)
     return (
       <div>
         <h5>{label} configuration</h5>
         <FormGroup>
           <Radio
-            checked={!useCustom}
+            checked={!useCustom && !useCustomUrl}
             name={name}
-            onChange={this._toggleCustomConfig}
+            onChange={(e) => this._toggleCustomConfig(e)}
             inline>
             Project default
           </Radio>
           <Radio
-            checked={useCustom}
+            checked={useCustom && !useCustomUrl}
```

Comment:
Is the !useCustomUrl necessary here?

Reply:
Yes as both can be true in which case we want to show the URL field